### ISSUE TITLE
The command docs is not work.The error caused by the argument of the function copyFileAsyn.

### ISSUE THREAD

**Issue by wangqi0902vip** [pixijs/pixi.js#4438]
I`ve tried the command of docs, and it was consoled an error said: binding.copyFile(src, dest, flags);
Error: EISDIR: illegal operation on a directory,  copyfile... 
I found that the arguments of the function copyFile was wrong, that one of them was a full name and the other was a directory name without extension name. It is work when i add file name for the toDir.
Hope that will help the others.

**Comment by bigtimebuddy** [comment]
Could you give more information about your environment. Specifically: OS version and Node.js version.

**Comment by wangqi0902vip** [comment]
OS:  OSX 10.13.1 
Node.js: v8.7.0
The error is under the folder of "pixi_jsdoc-template/jsdoc-template/publish.js" at line:374.

**Comment by bigtimebuddy** [comment]
Fixed with https://github.com/pixijs/pixi-jsdoc-template/commit/f6cbeca45d542806c444825113deb7387a7d6c09

**Comment by cwalther** [comment]
This change breaks things for me: it produces the hierarchy
```
out
  fonts
    glyphicons-halflings-regular.eot
      glyphicons-halflings-regular.eot
...
```
which is wrong and causes _Error: EEXIST: file already exists, mkdir 'doc/out/fonts/glyphicons-halflings-regular.eot/glyphicons-halflings-regular.eot'_ on the second run.

From what I can tell, this fix is wrong, it only works by accident on node >= v8.5 (vs. v6.11.2 here) and JSDoc v3.5.4, and the real fix is jsdoc3/jsdoc#1440 in JSDoc v3.5.5.

**Comment by cwalther** [comment]
Resolved by [pixi-jsdoc-template 2.4.2](https://github.com/pixijs/pixi-jsdoc-template/releases/tag/v2.4.2).
